packages/openid4vc/src/openid4vc-verifier/OpenId4VpVerifierService.ts

```diff
@@ -219,6 +224,7 @@ export class OpenId4VpVerifierService {
       response_mode: responseMode,
       response_type: 'vp_token',
       client_metadata,
+      verifier_attestations: options.verifierAttestations,
```

Comment:
Should we check that if `verifier_attestations` are provided we make sure that at least the `credential_ids` are present in the provided pex/dcql query?